stake-pool/js/src/index.ts

```diff
@@ -649,6 +649,58 @@ export async function withdrawSol(
   };
 }
 
+export async function addValidatorToPool(
+  connection: Connection,
+  stakePoolAddress: PublicKey,
+  validatorVote: PublicKey,
+  seed?: number,
+) {
+  const stakePool = await getStakePoolAccount(connection, stakePoolAddress);
+
+  const validatorList = await getValidatorListAccount(
+    connection,
+    stakePool.account.data.validatorList,
```

Comment:
nit: since you do this long property access a few times in here, why not simplify with something like this?

```ts
const stakePoolAccount = await getStakePoolAccount(connection, stakePoolAddress);
const stakePool = stakePoolAccount.account.data;
const { reserveStake, staker, validatorList } = stakePool;

const validatorListAccount = await getValidatorListAccount(connection, validatorList);
```

Reply:
Agreed and done